Header header
bool fly_flying
uint32 frame_count
sensor_msgs/Image image
